vendor.conf

```diff
@@ -15,6 +15,7 @@ golang.org/x/sys                        98c5dad5d1a0e8a73845ecc8897d0bd56586511d
 gopkg.in/check.v1                       20d25e2
 github.com/satori/go.uuid               5bf94b69c6b68ee1b541973bb8e1144db23a194b
 gopkg.in/yaml.v2			v2.2.2
+github.com/robfig/cron                  b41be1df696709bb6395fe435af20370037c0b4c
```

Comment:
Needs to add the vendor directory as well. Drone failed.

Reply:
Yes, will add and resend the PR